index.mjs

```diff
@@ -54,6 +54,7 @@ export function $(pieces, ...args) {
   let child = spawn($.prefix + cmd, options)
   let promise = new ProcessPromise((resolve, reject) => {
     child.on('exit', code => {
+      if(piped) process.stdin.unpipe(child.stdin)
```

Comment:
Why after exit? Not close?

Reply:
Oh yeah, after close is better.  I wasn't sure about the difference between the two.